### ISSUE TITLE
restrict_chat_member now working?

### ISSUE THREAD

**Issue by ticoti** [eternnoir/pyTelegramBotAPI#369]
Please answer these questions before submitting your issue. Thanks!

1. What version of pyTelegramBotAPI are you using?
Last one
2. What OS are you using?
ubuntu 12.10
3. What version of python are you using?
python 3.6

Have tried new version with restrict_chat_member and it simply doesnt work in any way I try, it does not restrict the user.

**Comment by alfilesnet** [comment]
Have you checked of your bot's permission?

I've tried to change the photo of a channel and it said me:
Bad Request: not enough rights to change chat photo

I changed it and now works perfectly.

**Comment by ticoti** [comment]
yes, it have permissions, it works with a simple  http api call  but not with this library and it is strange because it returns true, but it doesnt restrict the user





Enviado desde: ZUK Z2 Pro
El alfilesnet <notifications@github.com>, 1 jul. 2017 22:59 escribió:

Have you checked of your bot's permission?

I've tried to change the photo of a channel and it said me:
Bad Request: not enough rights to change chat photo

I changed it and now works perfectly.
